src/components/powerstrip.js

```diff
@@ -33,11 +33,15 @@ class MiniMediaPlayerPowerstrip extends LitElement {
   }
 
   get sourceSize() {
-    return (this.config.source === 'icon' || this.config.collapse || this.idle);
+    return (this.config.source === 'icon' || (this.config.hide.controls ^ this.config.hide.volume) || this.idle); // eslint-disable-line no-bitwise
```

Comment:
You should be able to do `(this.config.hide.controls !== this.config.hide.volume)` instead